units/units.hpp

```diff
@@ -2044,6 +2053,21 @@ namespace constants {
     constexpr precise_measurement a0{5.29177210903e-11, precise::m};
     /// Faraday's constant
     constexpr precise_measurement F = Na * e;
+
+    /// Josephson constant
+    constexpr precise_measurement Kj{483597.8484e9,
+                                     precise::Hz* precise::V.inv()};
+
+    /// magnetic flux quantum
+    constexpr precise_measurement phi0{2.067833848e-15, precise::Wb};
+
+    /// von Kiltzing constant
+    constexpr precise_measurement Rk{25812.80745, precise::ohm};
+
+    /// Rydberg constant
+    constexpr precise_measurement Rinf{10973731.568160,
+                                       precise::count / precise::m};
```

Comment:
Is `count/m` correct? I believe `count` is not the in the SI system, so would this not cause trouble?

(our applications use the `counts` unit a lot, so I believe this is really useful, but for better or for worse it is not ubiquitous)

Reply:
As I understand that constant it is expressed as cycles/unit distance.  I suppose I could define it in terms of radians/m but that would make the number look different.    The convert method can translate between rad/count and none fine, so at present I think it is the most appropriate to match up with the other defined units.

Reply:
You think I should change it?

Reply:
The way I understand this would be used is in some parts of equations, for example https://en.wikipedia.org/wiki/Rydberg_formula#For_hydrogen to compute a wavelength. If the resulting unit is `m/count` (or `nm/count`, ...) it will cause trouble in follow-up computations. Wavelength would expected to have a length unit.

Reply:
I actually think `m/count` would be more descriptive of a wavelength from a practical perspective, but I have never used that constant in any of my calculations so I will change it back to `1/m`